### PR TITLE
Create 1_file

### PR DESCRIPTION
**Pull request by nikita-awasthi**

### CODE REVIEW

File: file_2 [hunk 1]
```diff
@@ -0,0 +1 @@
+file 2
```
Comment (jai-bo) [inline file_2:1]: comment 1

**Review by jai-bo — CHANGES_REQUESTED** [review]

File: initial_commit_file [hunk 2]
```diff
@@ -0,0 +1 @@
+initial commit before pull request
```
Comment (nikita-awasthi) [inline initial_commit_file:1]: Some comment

File: new_branch_1_commit_after_pr [hunk 3]
```diff
@@ -0,0 +1 @@
+after pr
```
Comment (jai-bo) [inline new_branch_1_commit_after_pr:1]: Comment 1

Comment (jai-bo) [inline new_branch_1_commit_after_pr:1]: Comment 3 review

**Review by jai-bo — CHANGES_REQUESTED** [review]

File: new_branch_1_commit_after_pr (hunk 3 above)
[the remark this answers is not in the document]
Comment (nikita-awasthi) [reply new_branch_1_commit_after_pr:1]: owner nested comment

Comment (nikita-awasthi) [reply new_branch_1_commit_after_pr:1]: owner nested comment

Comment (jai-bo) [reply new_branch_1_commit_after_pr:1]: > owner nested comment

nested comment

**Comment by nikita-awasthi** [comment]
NA

**Pull request closed without merging**